src/reconcile/rec_write.c

```diff
@@ -1290,8 +1309,8 @@ __rec_txn_read(WT_SESSION_IMPL *session, WT_RECONCILE *r,
 		}
```

Comment:
Similarly, I disagree: since reconciliation has a transaction snapshot which is fixed while reconciliation is running, there is no chance of ambiguity when reading a transaction ID -- if it is visible as-of the snapshot, it is immutable (because committed transaction IDs are never rolled back).  If it is not visible as-of the snapshot, it doesn't matter whether reconciliation reads the original transaction ID or `WT_TXN_ABORTED` -- it was not be visible either way.